src/form/FileUpload/FileList/FileListStyles.ts

```diff
@@ -1,71 +1,71 @@
 import styled from 'styled-components';
-import { Button, Icon, themeColor, themeSpacing } from '@amsterdam/asc-ui';
+import { Button, themeColor, themeSpacing } from '@amsterdam/asc-ui';
+
+export const FileListTitleStyle = styled.h6`
+	align-self: baseline;
+	font-weight: 800;
+	font-size: 18px;
+	line-height: 130%;
```

Comment:
To my knowledge we're not using % line-height values anywhere. Why not pixels like we do elsewhere?